linux-ubuntu-2404-static.cmake

```diff
@@ -0,0 +1,53 @@
+
+if(DEFINED POLLY_CLANG_CXX17_STATIC_CMAKE_)
+  return()
+else()
+  set(POLLY_CLANG_CXX17_STATIC_CMAKE_ 1)
+endif()
```

Comment:
Please use include_guard() from now on these are from anotehter file and don't match the file name